app/Http/Controllers/CampController.php

```diff
@@ -159,6 +159,27 @@ public function store(Request $request, Validate $validate)
                 $message = trans('message.error.invalid_data');
                 return $this->resProvider->apiJsonResponse($status, $message, $result, $error);
             }
+            if(isset($request->parent_camp_num) && !empty($request->parent_camp_num)){
+                $parentCamp = Camp::where('camp_num',$request->parent_camp_num)->where('topic_num',$request->topic_num)->first();
```

Comment:
I think we can combine this sql statement with the $topic at line 165 using the left join